server/src/main/java/io/druid/client/cache/LocalCacheProvider.java

```diff
@@ -33,13 +40,60 @@
   @Min(0)
   private int initialSize = 500000;
 
-  @JsonProperty
-  @Min(0)
-  private int logEvictionCount = 0;
+  @JacksonInject
+  private DruidProcessingConfig config = null;
 
   @Override
   public Cache get()
   {
-    return new MapCache(new ByteCountingLRUMap(initialSize, logEvictionCount, sizeInBytes));
+    if(sizeInBytes > 0) {
+      return MapCache.create(sizeInBytes, initialSize, config.getNumThreads());
```

Comment:
From what I've read, for most use cases, setting a concurrency level to the number of threads should be enough, or do you have examples that show otherwise?